cgi-bin/ljlib.pl

```diff
@@ -54,6 +54,14 @@ BEGIN
     }
 }
 
+# Now set up logging support for everybody else to access
+use Log::Log4perl;
```

Comment:
All right. Looks like we will need to make sure Log::Log4perl gets installed before the code push, either in the dw user's extlib, or on each production system.  Otherwise the server won't even start, since you've added it to ljlib.pl.